rbd-target-api.py

```diff
@@ -2842,6 +2842,11 @@ def get_ssl_files_from_mon():
         # https://bugs.python.org/issue16487 is resolved, we should be able
         # to simply use file-like objects and makes this much nicer.
 
+        try:
+          data = data.encode('utf-8')
+        except:
+          pass
+
         tmp_f = tempfile.NamedTemporaryFile()
```

Comment:
Nit: since `pull_from_mon_config` always returns a string, perhaps just pass `mode='w'` (instead of the default 'w+b') to the `NamdTemporaryFile` constructor?

Reply:
This solution worked nicely. Changed the PR in favor of this option :+1: